app/src/main/java/org/greenstand/android/TreeTracker/di/NetworkModule.kt

```diff
@@ -41,4 +42,17 @@ val networkModule = module {
             .create(ApiService::class.java)
     }
 
+    single(named("paymentApi")) {
+        Retrofit.Builder()
```

Comment:
Can we try to reuse the http client we have for the other API calls? Also, let's try to keep these Module methods void of logic